docs/gc.md

```diff
@@ -35,31 +35,34 @@ To prevent applications running out of memory, objects in the Java heap that are
 
 ## Garbage collection policies
 
-OpenJ9 provides several different GC policies, each one specialized for operation with a specific heap configuration and one or more collectors. These policies are summarized in the followng table. Follow the links to read more about individual policies:
+OpenJ9 provides several GC policies that are designed around different application workloads. Each policy operates on a specific heap configuration and one or more collectors. These policies are summarized in the following table. Follow the links to read more about individual policies:
 
-| GC Policy                                        | Heap Configuration              | Collectors                         |
-|--------------------------------------------------|---------------------------------|------------------------------------|
-| [`gencon     `](#gencon-policy-default) (default)| Generational (nursery/tenure)   | Scavenger, Concurrent Mark, Global |
-| [`balanced   `](#balanced-policy)                | Generational (multiple regions) | Incremental Generational           |
-| [`metronome  `](#metronome-policy)               | Segregated                      | Realtime                           |
-| [`optavgpause`](#optavgpause-policy)             | Flat                            | Concurrent Mark, Global            |
-| [`optthruput `](#optthruput-policy)              | Flat                            | Global                             |
-| [`nogc       `](#nogc-policy)                    | --                              | --                                 |
+| GC policy name (option name)                                           |  Heap configuration             | Collectors                         |
+|------------------------------------------------------------------------|---------------------------------|------------------------------------|
+| [Generational concurrent (`gencon`)](#gencon-policy-default) (default) | Generational (nursery/tenure)   | Scavenger, Concurrent Mark, Global |
```

Comment:
* Mostly Concurrent Mark-Sweep (and optionally Compact) Global GC based on Incremental-update (Card-dirtying) write barrier
* Semi-space Evacuating Local GC (Scavenger). By default Stop-The-World, but optionally concurrent based on Loaded Valued Read Barrier

Reply:
I'm going to chat with Mel about the table as she has been working on how to fix it and I want to make sure this all lines up.

Reply:
Local or partial GC are more or less same thing. You can just pick one (or leave as is, no big deal). 
I do tend to define Scavenger as a local GC (operating locally under Nursery, fairly precisely defined part of heap), while we have PGC in Balanced, what really is Partial GC (operates under a subset of regions or part of heap, that are not completely predefined like Nursery is)

'on objects that are no longer in use' -> more precisely: 'on areas (or parts of) heap that yield most garbage (or where object die at high rate)'

Increments can be concurrent or STW also, not just full cycles:
'These cycles _or increments_ can also be concurrent '

'read or write barriers that are raised to allow application processing to continue without a STW collection taking place.' -> consider this: 'read or write object access barriers that are raised to allow GC operations run concurrently with application processing`

Reply:
Indeed, I had discussed this table with Mel. We can all discuss more to get some compromise between being correct and informative (as I was trying in my comments) on one side, but still fairly concise (since it's just a high level table) on the other side.